packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -67,16 +64,6 @@ class BuildIOSFrameworkCommand extends BuildSubCommand {
         help: 'Whether to produce a framework for the release build configuration. '
               'By default, all build configurations are built.'
       )
-      ..addFlag('universal',
-        help: '(deprecated) Produce universal frameworks that include all valid architectures.',
-        hide: !verboseHelp,
-      )
-      ..addFlag('xcframework',
```

Comment:
These are both deprecated, don't add to new command that will inherit from `BuildIOSFrameworkCommand`.